.github/workflows/pytest.yml

```diff
@@ -1,6 +1,7 @@
 name: Python CI
 
-on: [push]
+on:
```

Comment:
We need to add both push and pull request

```
on:
  - push
  - pull_request
```

Reply:
Okay I'll do that